src/FSharp.AWS.DynamoDB/Expression/ExpressionContainers.fs

```diff
@@ -28,116 +25,105 @@ type ConditionExpression<'TRecord> internal (cond : ConditionalExpression) =
     /// </summary>
-    static member Not(conditional : ConditionExpression<'TRecord>) =
+    static member Not(conditional: ConditionExpression<'TRecord>) =
         let qExpr = QueryExpr.ENot conditional.Conditional.QueryExpr
         ensureNotTautological qExpr
-        new ConditionExpression<'TRecord>(
-            { 
-                QueryExpr = qExpr
-                KeyCondition = extractKeyCondition qExpr
-                NParams = 0   
-            })
+        new ConditionExpression<'TRecord>({ QueryExpr = qExpr; KeyCondition = extractKeyCondition qExpr; NParams = 0 })
 
 /// Represents an update expression for a given record type
 [<Sealed; AutoSerializable(false)>]
-type UpdateExpression<'TRecord> internal (updateOps : UpdateOperations) =
-    let data = lazy(updateOps.GetDebugData())
+type UpdateExpression<'TRecord> internal (updateOps: UpdateOperations) =
+    let data = lazy (updateOps.GetDebugData())
     /// Internal update expression object
     member internal __.UpdateOps = updateOps
     /// DynamoDB update expression string
-    member __.Expression = let expr,_,_ = data.Value in expr
+    member __.Expression = let expr, _, _ = data.Value in expr
     /// DynamoDB attribute names
-    member __.Names = let _,names,_ = data.Value in names
+    member __.Names = let _, names, _ = data.Value in names
     /// DynamoDB attribute values
-    member __.Values = let _,_,values = data.Value in values
+    member __.Values = let _, _, values = data.Value in values
 
-    override __.Equals(other : obj) =
+    override __.Equals(other: obj) =
         match other with
         | :? UpdateExpression<'TRecord> as other -> updateOps.UpdateOps = other.UpdateOps.UpdateOps
         | _ -> false
 
     override __.GetHashCode() = hash updateOps.UpdateOps
 
-    static member (&&&) (this : UpdateExpression<'TRecord>, that : UpdateExpression<'TRecord>) =
-        UpdateExpression.Combine(this, that)
+    static member (&&&)(this: UpdateExpression<'TRecord>, that: UpdateExpression<'TRecord>) = UpdateExpression.Combine(this, that)
 
 and UpdateExpression =
     /// Combines a collection of compatible update expressions into a single expression.
-    static member Combine([<ParamArray>]exprs : UpdateExpression<'TRecord> []) =
+    static member Combine([<ParamArray>] exprs: UpdateExpression<'TRecord>[]) =
         match exprs with
         | [||] -> invalidArg "exprs" "must specify at least one update expression."
-        | [|expr|] -> expr
+        | [| expr |] -> expr
         | _ ->
 
-        let uops = exprs |> Array.collect (fun e -> e.UpdateOps.UpdateOps)
-        match uops |> Seq.map (fun o -> o.Attribute) |> tryFindConflictingPaths with
-        | None -> ()
-        | Some(p1,p2) ->
-            let msg = sprintf "found conflicting paths '%s' and '%s' being accessed in update expression." p1 p2
-            invalidArg "expr" msg
+            let uops = exprs |> Array.collect (fun e -> e.UpdateOps.UpdateOps)
```

Comment:
annoying that it does not allow trailing block to remain unindented
especially if it's going to leave the blank line

Reply:
There is an `fsharp_experimental_keep_indent_in_branch` option - looks like it just preserves the (un)indentation rather than moving it. I'll turn it on & see if I can find all the places it was used

Reply:
nice, looks like the right tool for the job per https://fsprojects.github.io/fantomas/docs/end-users/Configuration.html